hack/go.sh

```diff
@@ -64,39 +65,76 @@ test(){
   (
     cd "${REPO_ROOT}"
     for p in "${PKGS}"; do
-      go test "${p}"
+      go test -v "${p}"
     done
   )
 }
 
-preflight(){
+lint(){
+  (
+    cd "${REPO_ROOT}"
+    for p in "${PKGS}"; do
+      golangci-lint run "${p}"
+    done
+  )
+}
+
+linters(){
+  golangci-lint linters
+}
+
+ci-checks(){
     echo "-------- beginning preflight checks"
-    imports-check
-    vet
+    lint
     test
     build
 }
 
 help(){
-  msg=\
+  local msg=\
```

Comment:
this is fine but you can also not set a var and simply:
`` 
`cat <<EOF
...
EOF
```